tests/Bridge/Symfony/Bundle/Command/SwaggerCommandTest.php

```diff
@@ -11,27 +11,26 @@
 
 namespace ApiPlatform\Core\Tests\Bridge\Symfony\Bundle\Command;
 
-use Nelmio\ApiDocBundle\Tests\WebTestCase;
 use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
 use Symfony\Component\Console\Tester\ApplicationTester;
 
 /**
  * @author Amrouche Hamza <hamza.simperfit@gmail.com>
  */
-class SwaggerComandTest extends WebTestCase
+class SwaggerCommandTest extends KernelTestCase
 {
     public function testExecute()
     {
-        $this->getContainer();
+        self::bootKernel();
+
         $application = new Application(static::$kernel);
         $application->setCatchExceptions(false);
         $application->setAutoExit(false);
+
         $tester = new ApplicationTester($application);
```

Comment:
I think https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Tester/CommandTester.php is the right helper to use here.

For example, see https://github.com/symfony/symfony/blob/5129c4cf7e294b1a5ea30d6fec6e89b75396dcd2/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ConfigDebugCommandTest.php